test/api.spec.js

```diff
@@ -110,7 +114,10 @@ describe('ipfsd.api for Daemons', () => {
     it('check if API and Gateway addrs are correct', function (done) {
       this.timeout(30 * 1000)
```

Comment:
These timeouts are probably no longer needed (or less time) with shorter keysize. Please check all of them.